velox/docs/functions/spark/datetime.rst

```diff
@@ -13,6 +13,13 @@ These functions support TIMESTAMP and DATE input types.
     If num_days is a negative value then these amount of days will be
     deducted from start_date.
 
+.. spark:function:: date_diff(endDate, startDate) -> integer
+    Returns the number of days from startDate to endDate. 
```

Comment:
Would you generate the docs locally and make sure they look good. You may need to add an empty line here. CC: @kgpai

Reply:
Thanks! Just fixed it. I verified it  locally.
![image](https://github.com/facebookincubator/velox/assets/14065538/fbe4aa4b-b248-4aec-b1f3-5191085afa06)